src/components/RiveLogo.tsx

```diff
@@ -0,0 +1,24 @@
+import { useRive, useStateMachineInput } from '@rive-app/react-canvas'
+
+export default function () {
+  const stateMachineName = 'State Machine 1'
+  const { RiveComponent, rive } = useRive({
+    autoplay: true,
+    src: '/media/logo.riv',
+    stateMachines: stateMachineName,
+  })
+  const hoverHandler = useStateMachineInput(
+    rive,
+    stateMachineName,
+    'hover',
+    false
+  )
+
+  return (
+    <RiveComponent
+      alt="ketl"
+      onMouseEnter={() => hoverHandler && (hoverHandler.value = true)}
```

Comment:
No, I can' do `hoverHandler?.value = true`, it throws an error that you can't assign to probably non-existing value